raglweb/__init__.py

```diff
@@ -284,11 +285,38 @@ def player(profile_id):
 def info():
     return render_template('info.html', start_time=_cfg['start_time'])
 
+def _make_division_prefix(division_title):
+    """Convert the human readable division title into a prefix for use in the filename."""
+    words = division_title.upper().split(' ')
+    division_str = words[0]
+    if division_str.endswith('S'):
+        division_str = division_str[:-1]
+    division_str += ''.join([word[0] for word in words[1:] if len(word) > 0])
+    prefix = 'RAGL-S{:2d}-{}-'.format(_cfg['season'], division_str)
```

Comment:
I think you want `02d` here, otherwise it will insert a space.

BTW, you can also do:
```python
season = _cfg['season']
prefix = f'RAGL-S{season:02d}-{division_str}-'
```

But that's at your own discretion.

Reply:
Nice - I'd not seen the `f'...'` improvement before.